sound/soc/sof/sof-priv.h

```diff
@@ -39,6 +39,9 @@
 /* time in ms for runtime suspend delay */
 #define SND_SOF_SUSPEND_DELAY_MS	2000
 
+/* delay before scheduling D0i3 entry work */
+#define SND_SOF_D0I3_DELAY 5000
```

Comment:
should this be part of topology?

Reply:
It could be configured via topology file also, but I don't see the need for that yet, how about keeping macro here(can act as fallback timeout delay if no topology definition specified in the future).